scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingMethodImpl.java

```diff
@@ -157,6 +154,8 @@ public int schedule() {
         //get job Descriptor list with eligible jobs (running and pending)
         Map<JobId, JobDescriptor> jobMap = schedulingService.lockJobsToSchedule();
 
+        logger.debug("jobs selected to be scheduled : " + jobMap);
```

Comment:
Should be guarded with `if (logger.isDebugEnabled) { ... }` to prevent jobMap String to be created if debug is not enabled. It can really be an issue with 100k jobs in the queue.